metadata/contributors.json

```diff
@@ -5446,6 +5446,15 @@
          "macpherson"
       ]
    },
+   {
+      "emails" : [
```

Comment:
Plus you need to align this alphabetically and as per error, it should be above `Lorenzo Tilve` as below:

```
    {
       "emails" : [
         "lhernandez28@apple.com",
         "lizette4662@gmail.com"
      ],
      "github" : "liz4662",
      "name" : "Lizette Hernandez"
   },
```